mentat/command/commands/search.py

```diff
@@ -36,10 +41,19 @@ async def apply(self, *args: str) -> None:
                 if not await ask_yes_no(default_yes=True):
                     break
 
+    @override
+    @classmethod
+    def arguments(cls) -> List[CommandArgument]:
+        return [CommandArgument("required", "query")]
+
+    @override
     @classmethod
-    def argument_names(cls) -> list[str]:
-        return ["search_query"]
+    def argument_autocompletions(
+        cls, arguments: list[str], argument_position: int
+    ) -> list[str]:
+        return []
```

Comment:
Should search have the same autocompletes as normal prompt mode?

Reply:
Hmm good point; one thing that makes it a bit weird is that search is still a terminal command, so it's arguments are actually split up by space and processed with shlex (so quotes and backslashes will be treated weird). We could change it so only the search command just receives it's argument as the raw text it's user input, but that would be weird making an exception for only one command. Either way I like this idea and I think I'll implement it